exampleSite/config.toml

```diff
@@ -8,7 +8,6 @@ themesDir = "../.."
 enableGitInfo = true
 version = "0.13.0"
 disableKinds = ["RSS", "taxonomy", "taxonomyTerm"]
-paginate = 2
```

Comment:
What's the reason for removing paginate here?

Reply:
No reason actually but it was a priority fix so removing it asap seemed like a good idea. Should I move it to a patch PR?

Reply:
Was just a bit out of context. Let's try to do that in separate PRs to keep the context of PRs. We can leave it in here this time.